doc/releases/changelog-0.36.0.md

```diff
@@ -31,41 +31,117 @@
+          0.49756365+0.j, 0.47675786+0.j, 0.49756365+0.j, 0.47675786+0.j], requires_grad=True)
+  ```
 
-        # Apply the reflection
-        qml.Reflection(U)
+* Reflecting about a given quantum state is now available via `qml.Reflection`.
+  This operation is very useful in, say, the amplitude amplification algorithm.
+  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
 
-        return qml.state()
+  `qml.Reflection` works by providing an operation, :math:`U`, that *prepares* the 
+  desired state, :math:`\vert \psi \rangle`, that we want to reflect about. In other 
+  words, :math:`U` is such that :math:`U \vert 0 \rangle = \vert \psi \rangle`. In 
+  PennyLane, :math:`U` must be an `Operator`.
+  
+  For example, if we want to reflect about 
+  :math:`\vert \psi \rangle = \vert + \rangle`, then :math:`U = H`:
+
+  ```python
+  U = qml.Hadamard(wires=wires)
 
+  dev = qml.device('default.qubit')
+  @qml.qnode(dev)
+  def circuit():
+        qml.Reflection(U)
+        return qml.state()
   ```
 
   ```pycon
   >>> circuit()
-  tensor([1.+6.123234e-17j, 0.-6.123234e-17j], requires_grad=True)
+  tensor([0.-6.123234e-17j, 1.+6.123234e-17j], requires_grad=True)
   ```
+
+  For cases where :math:`U` comprises many operations, you can formulate it as an 
+  `Operator` by 
+  
+  * creating a user-defined 
+    [custom operation](https://docs.pennylane.ai/en/stable/development/adding_operators.html) 
+    with a decomposition, or by
+
+  * creating a quantum function containing each operation, one per line, then 
+    decorate the quantum function with `@qml.prod`, which turns the quantum 
+    function into a `Prod` instance:
+
+    ```python
+    @qml.prod
+    def U():
+        qml.Hadamard(wires=0)
+        qml.RY(0.1, wires=1)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Reflection(U())
+        return qml.state()
+
+    circuit()
+    ```
+
+    ```pycon
+    >>> circuit()
+    tensor([-0.00249792-6.13852933e-17j,  0.04991671+3.05651685e-18j,
+          0.99750208+6.10793866e-17j,  0.04991671+3.05651685e-18j], requires_grad=True)
+    ```
   
-* The `qml.AmplitudeAmplification` operator is introduced, which is a high-level interface for amplitude amplification and its variants.
+* A high-level interface for amplitude amplification and its variants is now 
+  available via the new `qml.AmplitudeAmplification` template.
   [(#5160)](https://github.com/PennyLaneAI/pennylane/pull/5160)
 
+  Based on [arXiv:quant-ph/0005055](https://arxiv.org/abs/quant-ph/0005055), 
+  given a state :math:`\vert \Psi \rangle = \alpha \vert \phi \rangle + \beta \vert \phi^{\perp} \rangle`, 
+  `qml.AmplitudeAmplification` amplifies the amplitude of :math:`\vert \phi \rangle`. 
+  Mathematically, 
+  :math:`\texttt{AmplitudeAmplification} \vert \Psi \rangle \approx \vert \phi \rangle`.
+
+  `qml.AmplitudeAmplification` requires
+
+  * `U`: an `Operator` that prepares :math:`\vert \Psi \rangle` (i.e., 
```

Comment:
Am trying a couple things to see if this gets fixed. No idea why this is happening 🤔

Reply:
POURQUOI

![Screenshot 2024-04-30 at 5 03 39 PM](https://github.com/PennyLaneAI/pennylane/assets/34751083/1d494f45-598a-490d-b40c-6c02ca840336)

Reply:
oh there's a typo 😆. Let's see what the fix brings...

Reply:
Still not showing up nicely for me :thinking: I don't have a suggestion on how to fix this though. We could just remove the "`qml.AmplitudeAmplification` requires" section, since they can look in the docstrings.